docs/upgrade-single-node-deployment.md

```diff
@@ -0,0 +1,120 @@
+If you want to upgrade a single node instance of XL Release or XL Deploy using Docker follow these steps,
+
+1. Make sure to read our docs for upgrade and release notes for each version you want to upgrade to.
+2. Stop current running docker container which have old version ythat need to be upgraded.
+3. Start new docker container with new tag which represent version you want to upgrade to, taking in consideration below
+    * include all volumes already being used with old docker images
```

Comment:
Need a note here that he cannot just mount the old conf volume, he has to modify the some conf files for the new version wheen applicable

Reply:
Okay will add this .. however i alreaday mentioned below,
Make sure to read our docs for upgrade and release notes for each version you want to upgrade to.

Reply:
yeah just be more specific so we do not miss it

Reply:
yep done